index.php

```diff
@@ -1 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My Cartoon Collection</title>
+</head>
+
+<body>
+
+<header>
+    <h1>My Cartoon Collection</h1>
+</header>
+
+
+
+
+
 <?php
+
+$db = new PDO('mysql:host=db;dbname=MyCollection', 'root', 'password'); // initialise the db connection
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT  `cartoons`.`character_name`, `TVshows`.`name`, `IQ`, `images` FROM `cartoons`
+JOIN `TVshows` ON `cartoons`.`TVshow_id` = `TVshows`.`id`;');
+$query->execute();
+$cartoons=$query->fetchAll();
+
+foreach ($cartoons as $character) {
+  $characterHtmlElement = '';
+  $characterNameElement = '<h2>' . $character['character_name'] . '</h2>';
```

Comment:
I think you'll need to add some headings here so that the IQ says IQ : 100 for example. at the moment it just shows as a number with no labelling.

Reply:
this was fixed in commit 47d9b3c (gave the wrong commit messages)